src/main/resources/assets/design-system/icons.scss

```diff
@@ -96,3 +96,19 @@
 .sci-icon-product-cart-blue {
   background-image: url(svgResource('/assets/images/icons/cart.svg', #3465CC));
 }
+
+.sci-icon-product-cart-gray {
+  background-image: url(svgResource('/assets/images/icons/cart.svg', #212529));
+}
+
+.sci-icon-download-gray {
+  background-image: url(svgResource('/assets/images/icons/download.svg', #212529));
+}
+
+.sci-icon-catalog-gray {
```

Comment:
Maybe `document`, `book` or `open-book` better shows what the icon really is.

Reply:
Just out of curiosity: shall colors be part of the icons or should it be added via extra classes?

Reply:
But we would have to rename the svg itself in that case, which could be breaking